libqtile/layout/floating.py

```diff
@@ -261,6 +261,10 @@ def info(self):
         d["clients"] = [c.name for c in self.clients]
         return d
 
+    def swap(self, client1, client2):
+        # This can't ever be called, but implement the abstract method
```

Comment:
This is not true actually, we could implement this but... I guess it's not very useful, unless we find a way to swap across layouts.